bee-network/bee-autopeering/src/discovery/manager.rs

```diff
@@ -749,19 +757,19 @@ pub(crate) async fn begin_verification(
         Ok(Ok(bytes)) => match VerificationResponse::from_protobuf(&bytes).map(|r| r.into_services()) {
             Ok(services) => Some(services),
             Err(e) => {
-                log::debug!("Verification response decode error: {}", e);
+                log::debug!("Verification response decode error for {}: {}", peer_id, e);
```

Comment:
This might be a bit clearer than just an `peer_id`:

```suggestion
                log::debug!("Verification response decode error for peer {}: {}", peer_id, e);
```

Same for the logging below.

Reply:
you're right. thx

Reply:
actually....I would have to edit a lot of logs if I follow through with this suggestion. Also the peer id cannot be taken for something other than a peer, so I guess there is not much benefit in terms of clarity. I'll keep it in mind though :slightly_smiling_face:

Reply:
Many of the other uses are ok I would say. It's just that this log does not have any information about where the error originated. That's why I thought it might be good to drop a hint there.